lib/services/s3.js

```diff
@@ -430,21 +430,6 @@ AWS.util.update(AWS.S3.prototype, {
     }
   },
 
-  /**
-   * @api private
-   */
-  computableChecksumOperations: {
-    putBucketCors: true,
-    putBucketLifecycle: true,
-    putBucketLifecycleConfiguration: true,
-    putBucketTagging: true,
-    deleteObjects: true,
-    putBucketReplication: true,
-    putObjectLegalHold: true,
-    putObjectRetention: true,
-    putObjectLockConfiguration: true
-  },
-
```

Comment:
Clear this out because it's not used anywhere.

Reply:
These properties aren't available